schematic_api/api/routes.py

```diff
@@ -546,8 +561,47 @@ def get_viz_tangled_tree_layers(schema_url, figure_type):
     layers = tangled_tree.get_tangled_tree_layers(save_file=False)
 
     return layers[0]
-    
-def download_manifest(input_token, dataset_id, asset_view, as_json, new_manifest_name=''):
+
+def download_manifest(input_token, manifest_id, new_manifest_name='', as_json=True):
+    """
+    Download a manifest based on a given manifest id. 
+    Args:
+        input_token: token of asset store
+        manifest_syn_id: syn id of a manifest
+        newManifestName: new name of a manifest that gets downloaded.
+        as_json: boolean; If true, return a manifest as a json. Default to True
+    Return: 
+        file path of the downloaded manifest
+    """
+    # call config_handler()
+    config_handler()
+
+    # use Synapse Storage
+    store = SynapseStorage(input_token=input_token)
+    # try logging in to asset store
+    try:
+        syn = store.login(input_token=input_token)
+    except SynapseAuthenticationError as e:
```

Comment:
do we need to catch these errors if they only just get raised after being caught?